python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -2496,7 +2496,7 @@ def test_report_runless_asset_events_permissions(
             variables={
                 "eventParams": {
                     "eventType": DagsterEventType.ASSET_MATERIALIZATION,
-                    "assetKey": {"path": ["asset1"]},
+                    "assetKey": {"path": ["asset_one"]},
```

Comment:
`asset1` is not in the graphql suite repositories so we early return here https://github.com/dagster-io/dagster/blob/master/python_modules/dagster-graphql/dagster_graphql/implementation/utils.py#L104

`asset_one` is so we exercise the location checks